zeek-ocsf/package.yaml

```diff
@@ -485,22 +485,24 @@ pipelines:
         protocol_name: "udp",
         protocol_num: 17,
       }
-      if zeek.client_addr.is_v6() or zeek.client_addr.is_v6() {
+      if not zeek.id?.dest_h? == null {
+        ocsf.connection_info.protocol_ver_id = 0
+      } else if zeek.id.orig_h.is_v6() {
         ocsf.connection_info.protocol_ver_id = 6
       } else {
         ocsf.connection_info.protocol_ver_id = 4
       }
       ocsf.src_endpoint = {
-        hostname: move zeek.host_name,
-        ip: move zeek.client_addr,
-        domain: move zeek.client_fqdn,
-        mac: move zeek.mac,
+        hostname: move zeek.host_name?,
+        ip: move zeek.client_addr? else zeek.assigned_addr?,
```

Comment:
Optional: a comment before the assignment would help explain why `assigned_addr` is a good alternative fit.